Source/Frontend/UI/UI.csproj

```diff
@@ -37,6 +37,9 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Update="Components\Engine Config\CorruptionEngineForm.cs">
```

Comment:
I have a feeling we're going to need to workaround in a bunch of places. Something to keep in mind if you ever have issues opening a Designer class in VS

Reply:
A bit annoying but VS is gonna VS.